bundles/binding/org.openhab.binding.zwave/database/widom/wps104.xml

```diff
@@ -0,0 +1,636 @@
+				conversely, it maintains its status (ON).</Help>
+			</Item>
+			<Item>	
+				<Value>2</Value>
+				<Label lang="en">OFF</Label>
+				<Help lang="en">If the initial status is ON the system switches to OFF: 
+				conversely, it maintains its status (OFF).</Help>
+			</Item>
+			<Item>	
+				<Value>3</Value>
+				<Label lang="en">TOGGLE</Label>
+				<Help lang="en">If the initial status is ON/OFF the system switches to OFF/ON.</Help>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>38</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Minimum>0</Minimum>
+			<Maximum>99</Maximum>
+			<Size>1</Size>
+			<Label lang="en">UP Power Associated</Label>
+			<Help lang="en">Sets the level of instantaneous power beyond which the time 
+			of permanence below this level is calculated. 
+			0 - Associated devices are switched OFF.
+			-1 - Associated devices are switched ON.
+			1 - 99 Set parameter for dimming purposes.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>39</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Minimum>0</Minimum>
+			<Maximum>11250</Maximum>
+			<Size>2</Size>
+			<Label lang="en">DOWN Power Level</Label>
+			<Help lang="en">Sets the level of instantaneous power beyond which the time 
+			of permanence below this level is calculated.
+			0 W - 11250 W Define level of instantaneous power.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>40</Index>
+			<Type>byte</Type>
+			<Default>10</Default>
+			<Minimum>0</Minimum>
+			<Maximum>10800</Maximum>
+			<Size>2</Size>
+			<Label lang="en">DOWN Power Time</Label>
+			<Help lang="en">SSets the time beyond which, if instantaneous power remains at 
+			levels below threshold defined by parameter 39, a DOWN Power event occurs.
+			0 sec - 10800 sec. Define time parameter.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>41</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label>DOWN Power State</Label>
+			<Help lang="en">Defines the next state of the device in the case of a DOWN Power event occurring.</Help>
+			<Item>	
+				<Value>0</Value>
+				<Label lang="en">IGNORE</Label>
+				<Help lang="en">The device maintains the initial status.</Help>
+			</Item>
+			<Item>	
+				<Value>1</Value>
+				<Label lang="en">ON</Label>
+				<Help lang="en">If the initial status is OFF the system switches to ON; 
+				conversely, it maintains its status (ON).</Help>
+			</Item>
+			<Item>	
+				<Value>2</Value>
+				<Label lang="en">OFF</Label>
+				<Help lang="en">If the initial status is ON the system switches to OFF: 
+				conversely, it maintains its status (OFF).</Help>
+			</Item>
+			<Item>	
+				<Value>3</Value>
+				<Label lang="en">TOGGLE</Label>
+				<Help lang="en">If the initial status is ON/OFF the system switches to OFF/ON.</Help>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>42</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Minimum>-1</Minimum>
+			<Maximum>99</Maximum>
```

Comment:
No - either, or... In theory, I think it should be 255 since I don't think negative values are (in theory!) allowed in zwave. However, lots of devices manuals define it this way, and the binding mostly doesn't care (I think!).